ckanext/gdi_userportal/scheming/schemas/solr_schema.xml

```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need to think of migration in case fields are to be added in the future? For now if I do not remove everything and just re-build it fails on re-index

Reply:
Very good question. Most probably yes and no. We need to test and understand how SOLR works, but this is out of scope of this PR.

What happened to me was: while I was debugging, I encouter things to change/fix. I did the changes in the schema file and rebuilt/redeployed the image without removing the volume. After that, I executed `ckan -c ckan.ini search-index rebuild`. 

I did not have any issue, but this may not be de case all the time.